tests/test_cli_compile.py

```diff
@@ -513,40 +512,34 @@ def test_locally_available_editable_package_is_not_archived_in_cache_dir(
 @pytest.mark.parametrize(
     ("line", "dependency"),
     (
-        # zip URL
         # use pip-tools version prior to its use of setuptools_scm,
         # which is incompatible with https: install
-        (
+        pytest.param(
```

Comment:
Added `pytest.param` to add an id to the tests, I think it's much nicer and easier to follow when we need to debug.